inc/class-core-sitemaps-stylesheet.php

```diff
@@ -503,4 +479,42 @@ protected function get_stylesheet_css() {
 		 */
 		return apply_filters( 'core_sitemaps_stylesheet_css', $css );
 	}
+
+	/**
+	 * Get the lookup table for column headings to be used in the sitemap stylesheet.
+	 *
+	 * @return string
+	 */
+	protected function get_stylesheet_column_headings() {
+		$column_headings = array(
+			'http://www.sitemaps.org/schemas/sitemap/0.9' => array(
+				'loc'        => esc_html__( 'URL', 'core-sitemaps' ),
+				'lastmod'    => esc_html__( 'Last Modified', 'core-sitemaps' ),
+				'changefreq' => esc_html__( 'Change Frequency', 'core-sitemaps' ),
+				'priority'   => esc_html__( 'Priority', 'core-sitemaps' ),
+			)
+		);
+		/**
+		 * Filter the column headings used in the sitemap stylesheet.
+		 *
+		 * @param array $column_headings Keys are namespace URIs and values are
+		 *                               arrays whose keys are local names and
+		 *                               whose values are column headings.
+		 */
+		$column_headings = apply_filters( 'core_sitemaps_stylesheet_column_headings', $column_headings );
```

Comment:
Here's an example how to use this new filter with the current state of this plugin (i.e., when extension elements are in the `http://www.sitemaps.org/schemas/sitemap/0.9` namespace):

```
add_filter( 'core_sitemaps_posts_url_list', function( $url_list ) {
   foreach ( $url_list as &$url_item ) {
      $url_item['my-extension-element' = __( 'some value', 'my-plugin' );
   }

   return $url_list;
} );

add_filter( 'core_sitemaps_stylesheet_column_headings', function( $column_headings ) {
   $column_headings['http://www.sitemaps.org/schemas/sitemap/0.9']['loc']                  = __( 'Permalink', 'my-plugin' );
   $column_headings['http://www.sitemaps.org/schemas/sitemap/0.9']['my-extension-element'] = __( 'Cool custom element', 'my-plugin' );
} );
```

And here's how it would be used if/when somethink like the proposal in https://github.com/GoogleChromeLabs/wp-sitemaps/issues/151#issuecomment-612252959 is incorporated in this plugin:

```
add_filter( 'core_site_maps_namespace_bindings', function( $namespace_bindings ) {
	$namespace_bindings[ 'my-plugin' ] = 'urn:my-plugin';
} );

add_filter( 'core_sitemaps_posts_url_list', function( $url_list ) {
   foreach ( $url_list as &$url_item ) {
      $url_item['my-plugin:extension-element' = __( 'some value', 'my-plugin' );
   }

   return $url_list;
} );

add_filter( 'core_sitemaps_stylesheet_column_headings', function( $column_headings ) {
   $column_headings['http://www.sitemaps.org/schemas/sitemap/0.9']['loc'] = __( 'Permalink', 'my-plugin' );
   $column_headings['urn:my-plugin]['extension-element']                  = __( 'Cool custom element', 'my-plugin' );
} );
```